setup/README.md

```diff
@@ -21,7 +21,7 @@ A guide to setting up your Raspberry Pi
 
 ### Optional
 
-- Ethernet (network) cable [Model B/B+/2/3 only]
+- Ethernet (network) cable [Model B/B+/2/3B/3B+ only]
```

Comment:
For consistency, probably wouldn't hurt to change "2" to "2B"? :stuck_out_tongue_winking_eye:

Reply:
Wondered about that, but we didnt release a 2A, so not sure.

Reply:
I'll leave it up to the copy-editors.